doc/help.rst

```diff
@@ -37,6 +53,23 @@ This can be useful when docstrings are not available or insufficient to document
 
 |
 
+===========================
+Getting help through hovers
+===========================
+
+You can also get help from objects through hovers, which are shown over any method, function or class when placing the cursor on top of it. Clicking the hover will open the information of the object in the Help panel.
+
+.. image:: images/help/help_hover.png
+
+|
+
+===========================
+Deactivate automatic import
+===========================
+
+When you get help in the IPython console from an object that has not been previously imported, the object is automatically imported in the internal console so that documentation can be shown when available. This can be disabled in the options menu in the Help panel so that only documentation from imported objects is displayed.
```

Comment:
```suggestion
When you get help in the :guilabel:`IPython Console` for an object that has not been previously imported, it is automatically loaded in the :doc:`internalconsole` so that documentation can be shown when available.
This can be disabled in the :guilabel:`Help` pane's top-right options menu so that only documentation from imported objects is displayed.
```

* Correct/add a large amount of required formatting
* Avoid repetition
* Fix grammar/phrasing
* Improve clarity
* Consistent terminology (panel/pane)